src/rpc/rpc-handlers.ts

```diff
@@ -181,30 +183,9 @@ const specialHandlers: Record<string, HandlerFunc> = {
             return {status: statusCode, result: {code: statusCode, message: err.message}};
         }
     },
-    keplr: async (data) => {
-        const {method, params} = data;
-        const methodName = method.split('_')[1];
-        // eslint-disable-next-line @typescript-eslint/ban-types
-        const keplrMethod = keplr[methodName as keyof Keplr] as Function;
-        if (typeof keplrMethod === 'function') {
-            try {
-                const result = await keplrMethod.apply(keplr, params) || {};
-                return {status: 200, result};
-            } catch (error) {
-                return {
-                    status: 400, result: {
-                        __error: {
-                            message: error.message,
-                            module: error.module,
-                            code: error.code
-                        }
-                    }
-                };
-            }
-
-        }
-        return {status: 500, result: 'Not valid method'};
-    }
+    // Keplr
+    keplr_sendTx: storeTransaction,
+    keplr_confirmTx: confirmTransaction
```

Comment:
Here we _intercept_ the transactions that cannot be immediately sent to the blockchain. For now it's only `sendTx` but we might need to add others depending on whether they need user confirmation or not.

`keplr_confirmTx` is a made up tx name that we can use to detect user confirmation. In reality, this is not strictly needed as all confirmations are coming in as `eth_confirmTransaction` and then a network check is done in the `confirmTransaction` handler. I think this can be improved and we shouldn't have done it like this when Solana support was added, but it is out of scope for keplr integration and I suggest not to worry about it now.

Reply:
is not sendTx which requires confirmation but signTx